.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: release
```

Comment:
This will not work right now since goreleaser will miss the credentials to push to dockerhub. If you want to merge this PR quickly, we can remove this publish WF from this PR and we merge it without it. We can add publish in a new PR

Reply:
I will remove the release workflow from this PR for now.